charts/nginx-gateway-fabric/README.md

```diff
@@ -245,51 +252,56 @@ kubectl kustomize https://github.com/nginxinc/nginx-gateway-fabric/config/crd/ga
+| nginxGateway.leaderElection.lockName | string | Autogenerated if not set or set to "". | The name of the leader election lock. A Lease object with this name will be created in the same Namespace as the controller. |
+| nginxGateway.lifecycle | object | `{}` | The lifecycle of the nginx-gateway container. |
+| nginxGateway.productTelemetry.enable | bool | `true` | Enable the collection of product telemetry. |
+| nginxGateway.readinessProbe.enable | bool | `true` | Enable the /readyz endpoint on the control plane. |
+| nginxGateway.readinessProbe.initialDelaySeconds | int | `3` | The number of seconds after the Pod has started before the readiness probes are initiated. |
+| nginxGateway.readinessProbe.port | int | `8081` | Port in which the readiness endpoint is exposed. |
+| nginxGateway.replicaCount | int | `1` | The number of replicas of the NGINX Gateway Fabric Deployment. |
+| nginxGateway.securityContext.allowPrivilegeEscalation | bool | `false` | Some environments may need this set to true in order for the control plane to successfully reload NGINX. |
+| nodeSelector | object | `{}` | The nodeSelector of the NGINX Gateway Fabric pod. |
+| service.annotations | object | `{}` | The annotations of the NGINX Gateway Fabric service. |
+| service.create | bool | `true` | Creates a service to expose the NGINX Gateway Fabric pods. |
+| service.externalTrafficPolicy | string | `"Local"` | The externalTrafficPolicy of the service. The value Local preserves the client source IP. |
+| service.ports | list | `[{"name":"http","port":80,"protocol":"TCP","targetPort":80},{"name":"https","port":443,"protocol":"TCP","targetPort":443}]` | A list of ports to expose through the NGINX Gateway Fabric service. Update it to match the listener ports from your Gateway resource. Follows the conventional Kubernetes yaml syntax for service ports. |
+| service.type | string | `"LoadBalancer"` | The type of service to create for the NGINX Gateway Fabric. |
+| serviceAccount.annotations | object | `{}` |  |
+| serviceAccount.imagePullSecret | string | `""` | The name of the secret containing docker registry credentials. Secret must exist in the same namespace as the helm release. |
+| serviceAccount.imagePullSecrets | list | `[]` | A list of secret names containing docker registry credentials. Secrets must exist in the same namespace as the helm release. |
+| serviceAccount.name | string | Autogenerated if not set or set to "" | The name of the service account of the NGINX Gateway Fabric pods. Used for RBAC. |
+| terminationGracePeriodSeconds | int | `30` | The termination grace period of the NGINX Gateway Fabric pod. |
+| tolerations | list | `[]` | Tolerations for the NGINX Gateway Fabric pod. |
+
```

Comment:
Is it possible to configure the tool to shrink down the table a bit? It's very wide and requires some scrolling. The `default` column is the main issue due to one or two wide fields.

Reply:
Unfortunately it's not customizable 🙁 

I think the biggest problem is `service.ports` that is a big object. We could split it up for every entry and like document what the name is, the port, etc..

Reply:
Yeah I think that would be good

Reply:
That's probably overkill on second thought (deleted my last comment).

Reply:
I found out that it's possible to output HTML for the table. This would give us more control but I'm not sure if we want mixed content.